pkg/network/bootstrap.go

```diff
@@ -0,0 +1,40 @@
+package network
+
+import (
+	"github.com/pkg/errors"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/gophercloud/utils/openstack/clientconfig"
+
+	operv1 "github.com/openshift/api/operator/v1"
+)
+
+type KuryrBootstrapData struct {
+	ServiceSubnet     string
+	PodSubnetpool     string
+	WorkerNodesRouter string
+	WorkerNodesSubnet string
+	PodSecurityGroups []string
+	ClusterID         string
+	OpenStackCloud    clientconfig.Cloud
+}
+
+type BootstrapData struct {
```

Comment:
BootstrapData is the *result* of bootstrapping, right? Then I'd rather be called something like `BootstrapResult` or `BootstrapStatus`. Thoughts?

Reply:
Makes sense to me, changing now.